eZ/Publish/API/Repository/Tests/SearchService/SearchServiceFulltextEmbedTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests\SearchService;
+
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchHit;
+use eZ\Publish\Core\FieldType\RichText\Value as RichTextValue;
+use eZ\Publish\Core\Repository\Values\Content\Content;
+use eZ\Publish\Core\Repository\Values\Content\ContentCreateStruct;
+
+/**
+ * Test case for full text search in the SearchService (for embed).
+ *
+ * @see \eZ\Publish\API\Repository\SearchService
+ * @group integration
+ * @group search
+ * @group fulltext
+ */
+class SearchServiceFulltextEmbedTest extends BaseTest
```

Comment:
naming:
```suggestion
class SearchServiceFullTextEmbedTest extends BaseTest
```

Reply:
I treat a Fulltext / Full-text as one phrase, so current class name looks good in my opinion and is consistent with SearchServiceFulltextTest

Reply:
> I treat a Fulltext / Full-text as one phrase, so current class name looks good in my opinion and is consistent with SearchServiceFulltextTest

I'm sorry, but in this case it doesn't matter. In the eZ Platform domain, any piece of code related to full-text search has the only one correct form: `FullText` (in writing/doc - it depends). Please notice that that "FullText" is consistent with the Criterion name used by the test.

The original "SearchServiceFulltextTest" is a technical debt.